src/Microsoft.ML.Data/Evaluators/Metrics/AnomalyDetectionMetrics.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Microsoft.Data.DataView;
+
+namespace Microsoft.ML.Data.Evaluators.Metrics
+{
+    public sealed class AnomalyDetectionMetrics
+    {
+        public double Auc { get; }
```

Comment:
Summaries, Remarks, and links to relevant documentation.

Reply:
added basic summaries for now.

wanted to also add the remarks from TLC website., but the explanations there were not clear esp. for the detection rate metrics.  

---
In reply to: [255583277](https://github.com/dotnet/machinelearning/pull/2390#discussion_r255583277) [](ancestors = 255583277)

Reply:
For these summaries, check in with @shmoradims ; he's building a set of generic docs for things like AUC, F1, RMSE, etc.

---
In reply to: [255703503](https://github.com/dotnet/machinelearning/pull/2390#discussion_r255703503) [](ancestors = 255703503,255583277)